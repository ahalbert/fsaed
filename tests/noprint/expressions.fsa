1: /target/ ->
2: {let count = 4 rewind /div/ -> }
3: {let count = count - 1 if count == 0  { -> } else { rewind /div/ } }
4: { let count = 5 start capture myvar -> } 
5: { println count /div/ let count = count - 1 if count == 0 -> }
6: {print myvar ->}
