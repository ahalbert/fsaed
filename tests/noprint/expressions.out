5
4
3
3
2
1
  <div id="this two">
    <div id="this three">
      <div id="this four">
      target
      </div>
    </div>
  </div>
